reader/index.html

```diff
@@ -11,35 +12,41 @@
 
 <body>
 
-	<button id="settings-button" class="fa fa-gear" title="Reader Settings"></button>
+	<button id="settings-button" class="fa fa-gear" data-label="buttonReaderSettings"></button>
 	<div id="settings-dropdown" hidden>
+		<label data-string="themeReader"></label>
```

Comment:
I'm not sure this needs a label; I think having the colored circles is enough to figure out what they do.

Reply:
the intention is more in the future to add the option to change the font color, I am already introducing the theme indicator

Reply:
What are some examples of situations where you would want to change the font color? I can't think of any combinations that work well except white-on-black or black-on-white (or almost-black on sepia), and we already have those.

Reply:
I think the choice of color is a lot of the person’s taste, particularly use links in the same color as the text, to avoid distractions, there may be other people who like something like that

Reply:
I'm not sure I agree; I think what we have is a pretty good balance between having the links not stand out too much, and still being able to tell they're there. In most articles that I've seen with links, the links are typically useful and an important part of the article, so I can't think of a lot of cases where I'd want the links to be hidden completely.

Reply:
ok, it can be removed, i make a path for my use, do you know any automatic way to remove this item from translations? I put file by file, it's sad to think about removing one at a time kkkkkkk

Reply:
VS Code supports global find-and-replace with regexes, so I removed it with this: `\t"themeReader":.*\n`.